config/Symphony.config

```diff
@@ -16,6 +16,9 @@
     },
     "customFlags": {
         "authServerWhitelist": "",
-        "authNegotiateDelegateWhitelist": ""
+        "authNegotiateDelegateWhitelist": "",
+        "disableGpu": false,
```

Comment:
do we need such granular configuration?  
that is, are there cases where we just need to disable one but not others?

Reply:
agreed. makes sense just to have one attribute. will push the change.